src/components/Login/Login.js

```diff
@@ -189,6 +190,8 @@ class Login extends Component {
       } else {
         this.setState({ loaded: true });
       }
+    } else {
+      this.setState({ loaded: true });
```

Comment:
when server give 200 and no json...
Actually  it is some error state, but we should handle any unexpected state.